examples/unattended_baggage/demo.ipynb

```diff
@@ -0,0 +1,330 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "select_cons = {\n",
+    "    \"track_id\": None,\n",
+    "    \"tlbr\": lambda x: str(x),\n",
+    "    \"no_owner_periods\": None,\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The query is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class FindUnattendedBaggage(vqpy.QueryBase):\n",
+    "    @staticmethod\n",
+    "    def setting() -> vqpy.VObjConstraint:\n",
+    "        return vqpy.VObjConstraint(\n",
+    "            filter_cons=filter_cons,\n",
+    "            select_cons=select_cons,\n",
+    "            filename=\"unattended_baggage\",\n",
+    "        )"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 3: Running the query"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the `Person` VObj and the query defined, we can run the query, where:\n",
+    "\n",
+    "- `cls_name` is a tuple for mapping numerical outputs of object detector to str\n",
+    "\n",
+    "\te.g. the built-in `vqpy.COCO_CLASSES` here starts with `(\"person\", \"bicycle\", ...)`, meaning that we will map output `0` of object detector to COCO class `\"person\"`\n",
+    "\n",
+    "- `cls_type` is a dictionary that maps name of object type (in str) to VObj types defined\n",
+    "\n",
+    "\t`{\"person\": Person, \"backpack\": Baggage, \"suitcase\": Baggage}` means we wish to map COCO class `person` to VObj type `Person`, and `backpack, suitcase`  to `Baggage`\n",
+    "\n",
+    "- `tasks` is a list of queries to run on the video"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vqpy.launch(\n",
+    "    cls_name=vqpy.COCO_CLASSES,\n",
+    "    cls_type={\"person\": Person, \"backpack\": Baggage, \"suitcase\": Baggage},\n",
+    "    tasks=[FindUnattendedBaggage()],\n",
+    "    video_path=video_path,\n",
+    "    save_folder=save_folder,\n",
+    "    detector_model_dir=detector_model_dir,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Expected result\n",
+    "\n",
+    "Result of the query will be in `{save_folder}/{video_name}_{task_name}_{detector_name}.json`, output for this example should be in `./vqpy_outputs/baggage_unattended_baggage_yolox.json`.\n",
+    "\n",
+    "One entry is created for each frame that has filter condition satisfied.\n",
+    "\n",
+    "At 3102th frame (around 103s), we have output:\n",
```

Comment:
Explain why you choose this frame. like we detected there is no owner for the baggage in frame of 93s, the no owner condition continues for 10s, so in the 103s frame, we can see that the unattended baggage id in the output, which is the baggage below.

Reply:
Makes sense, explanation added.